tests/fxl-properties-001/OPS/package.opf

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout Properties</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-09-22</dc:date>
+    <dc:description>
+    When the rendition:layout property is set to pre-paginated, Reading Systems MUST NOT include space between the adjacent content slots when rendering Synthetic Spreads.
```

Comment:
So far we have been writing the tests in this format, not as normative statements:
https://github.com/w3c/epub-tests/blob/main/tests/cmt-gif/OPS/package.opf

I think we could do either, but we should be consistent. Reword to match, or discuss at next test meeting?

Reply:
I'll reword to match!